### ISSUE TITLE
Inconsistent Coloring Patterns for Optional Options

### ISSUE THREAD

**Issue by emmaagcap** [curvenote/support#12]
Optional options in Export Template Options Page 1 are colored orange when the required options are red. When the required options are then green, the optional options are also green (even if they have not been selected/changed).

Optional options in Export Template Options Page 2 - tagged blocks are colored orange independent of the inclusion of required options (abstract) - at this point there is not a check of "summary" or "acknowledgement" tags so this may be linked.

Should be consistent for these options - maybe always orange unless all optional options are selected/used? - then go green.

![image](https://user-images.githubusercontent.com/89223352/138761621-36ade807-c4ed-4984-baf5-4fa856516371.png)
![image](https://user-images.githubusercontent.com/89223352/138761670-b51865ea-a45f-418e-90bc-0ae509741871.png)

**Comment by stevejpurves** [comment]
Yes, I see what you mean. The issue with the second image that you show here (page 1) is that Draft being true / false is actually in a valid state (of false) so it has been specified.

Only turning the Optional Green when Draft is true could also be confusing. I think the same issue occurs with any Boolean option.

Maybe coloring should be changed such that the `Optional N` is only plain/black unless there is an invalid optional field, where it then goes red?

**Comment by emmaagcap** [comment]
What about:

Required:
-  red "triangle!" next to all required fields that corresponds to the number of red Required: # at the top of the page.
- when required field is completed red "triangle!" is replaced with a green checkmark
- required: # will turn green when all have been completed

Optional
- orange "triangle!" next to all optional fields.
- when optional fields are completed the orange "triangle!" is replaced with a green checkmark
- optional: # at the top of the page always black unless ⬇️ 
- if an optional option is incomplete OR is an issue the optional: # is marked red (cannot proceed)
- the optional option that has an issue now has a red "triangle!" tag.

might be too confusing tho but better to be safe and check all the details for users.

**Comment by stevejpurves** [comment]
It's difficult to maintain consistency between the form and the tagged block "report" for mechanical reasons.
We'll continue to improve this but one small change has been made to remove the coloring of the `Optional N` heading at the top of the dialog.
